cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -45,7 +45,7 @@ spec:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster-amd64:v1.5.0-beta.0
+        - image: k8s.gcr.io/heapster-amd64:v1.5.0-beta.0
           name: heapster
```

Comment:
I am not certain how heapster is released, but I am guessing they will need updates in there repo as well?

Reply:
Both names will work for forseeable future.